com.github.sdbg.debug.ui/src/com/github/sdbg/ui/ISDBGLauncher.java

```diff
@@ -0,0 +1,11 @@
+package com.github.sdbg.ui;
+
+import org.eclipse.core.resources.IResource;
+
+public interface ISDBGLauncher {
+
+  String EXTENSION_ID = "com.github.sdbg.debug.ui.sdbgLauncher";
+
+  public void launchChrome(IResource resource, String mode, String url);
```

Comment:
Good question. I looked and didn't see any yet. The methods were protected in the ChromeLaunchShortcut as it was meant for a purpose inherited by its super class. For the extension point, I thought the readability of having it's own type might be preferred although is some cases I see the interface as a inner interface type for the concrete class. I didn't do a 1 to 1 interface for the shortcut as I thought it might be to restrictive b/c the logic seemed as if if should be baked into a utility that is more multipurpose.